index.js

```diff
@@ -1,30 +1,66 @@
+// Wait, what do we do for this one?
+// I'm going to go ahead and work on the suggestions thing... This is AmazingMech2418 by the way
+// Okay. Suggestions work. I did a method with no client-side JS, but we may need it if we want to improve the UI. We need a DB though. Currently, it just logs to the console. Also maybe Repl Auth?
+
+// Whippingdot - Bro We need the suggestion to output in a file or a db. Console is fine for now but we need to work on making it output in the file. Also Repl auth is a feature we need, I will try to make it work.
+
+/**
+ * gosh the css sucks
+ * and what is this project for?
+ * and what should i do?
+ * -rohil
+ * Whippingdot: CSS is great
+ * this project is for repl.it
+ * you have to make the suggestions be sent to a db or file, you have to make the suggestions page show the name of person and their suggestion below the form, and you have to make it so one suggestions per person. You will now the person cause of repl auth
+ **/
+
 const express = require("express");
 const app = express();
 
 // Finally got this line to work, so /static leads to any necessary static content. ~ SixBeeps
-app.use('/static', express.static(__dirname + '/static'));
-app.use('favicon.ico', express.static(__dirname + '/favicon.ico'))
```

Comment:
Why is this here? Can't you just move the favicon into the static folder?

Reply:
I don't know, there were some glitches and stuff wasn't working. You know you and @firefish111 and anyone really can change the project however they want. Just make sure you revert changes if something doesn't work. Stop requesting changes. Also don't review changes, review the changes when this pr is done, we aren't done with the pr yet.

Reply:
@whippingdot You can do some comment isolation.
Basically comment out some things that people say won't work, and if it doesn't work, then uncomment that line out a comment something else around it. It's a very good strategy for bug-fixing.

Reply:
Ok, thanks for the help